bin/phpcs-excludelist.json

```diff
@@ -589,15 +574,11 @@
 	"tests/php/sync/test_interface.jetpack-sync-replicastore.php",
 	"tests/php/test_class.functions.opengraph.php",
 	"tests/php/test_class.jetpack-idc.php",
-	"tests/php/test_deprecation.php",
 	"tests/php/test_functions.photon.php",
-	"tests/php/uninstall/test_uninstall.php",
+	"tests/php/test_deprecation.php",
```

Comment:
Alphabetical order, please. If you switch these two lines around the diff won't show an addition.

Reply:
Resolved, the uninstall/test_uninstall.php shouldn't be listed anymore.